opsdroid/connector.py

```diff
@@ -71,3 +71,14 @@ def __init__(self, config):
 
         """
         raise NotImplementedError
+
+    async def disconnect(self, opsdroid):
```

Comment:
Should this method have a `pass` after the docstring as it (rightly) doesn't `raise`?

Reply:
I can add one to be explicit, but it's not required (just the docstring is sufficient).

Reply:
Explicit is good as it makes the code more readable.

Reply:
if we were being really explicit about it, it should be `return None` 😉

Reply:
True! I'll leave it up to you. I'm aiming for readability.